### PR TITLE
feat(#302): Add new get Ldevice Status method

### PR DESCRIPTION
**Pull request by gleizesDor**

### CODE REVIEW

**Comment by samirromdhani** [comment]
How do I correctly review this?  RSR-770 it does not appear to be the right issue.
It would be better to connect it to the appropriate issue and give the pull request a nice English name ;)

File: sct-commons/src/main/java/org/lfenergy/compas/sct/commons/util/LdeviceStatusEnum.java [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+package org.lfenergy.compas.sct.commons.util;
+
+import lombok.Getter;
+
+import java.util.Arrays;
+
+/**
+ * Object describing Ldevice Status, prefere this to LdeviceStatus constants
+ */
+public enum LdeviceStatusEnum {
```
Comment (samirromdhani) [inline sct-commons/src/main/java/org/lfenergy/compas/sct/commons/util/LdeviceStatusEnum.java:14]: A class called [LDeviceStatus](/com-pas/compas-sct/blob/develop/sct-commons/src/main/java/org/lfenergy/compas/sct/commons/util/LDeviceStatus.java#L12) already exists. You can use or delete it and keep the one you created.

Comment (gleizesDor) [reply sct-commons/src/main/java/org/lfenergy/compas/sct/commons/util/LdeviceStatusEnum.java]: Removed

**Comment by samirromdhani** [comment]
The branch name needs to be adjusted ~~feat/RSR-770_Refacto_LDGRPService_configureLDGRP~~

**Review by samirromdhani — APPROVED** [review]
The branch name needs to be adjusted ~~feat/RSR-770_Refacto_LDGRPService_configureLDGRP~~

**Pull request merged**